manim/camera/mapping_camera.py

```diff
@@ -67,6 +81,13 @@ def capture_mobjects(self, mobjects, **kwargs):
 
 # TODO, the classes below should likely be deleted
 class OldMultiCamera(Camera):
+    """Parameters
+    ----------
+    *cameras_with_start_positions : tuple
```

Comment:
IIRC it is recommended to document starred args / kwargs without including the asterisks.

```suggestion
    cameras_with_start_positions : tuple
```

Reply:
In general, we're not including them in our docstrings. However, [our docs](https://docs.manim.community/en/stable/contributing/docs/docstrings.html) say that we're following the [NumPy style guide](https://numpydoc.readthedocs.io/en/latest/format.html#parameters), which does recommend to include the leading stars. In the future, there should probably be a PR to make our docstrings conform more to that style guide.

Reply:
I'll remove it here and we should have a discussion on how to handle this globally throughout our code base.